<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pointSelDlg</class>
 <widget class="QDialog" name="pointSelDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>转发插件</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>发送周期设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>遥测周期：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ycCycleEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>秒</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>遥信周期：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="yxCycleEdit"/>
            </item>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>秒</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>通讯设置</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>串口设置：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="serialPortComboBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>  波特率：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="baudComboBox">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>23</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="transTabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>遥信设置</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTableWidget" name="yxTableWidget">
           <column>
            <property name="text">
             <string>点号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>测点名称</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>遥测设置</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <item>
          <widget class="QTableWidget" name="ycTableWidget">
           <column>
            <property name="text">
             <string>点号</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>测点名称</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
